internal/logging/file.go

```diff
@@ -112,64 +108,6 @@ func (l *fileHandler) Emit(ctx *MessageContext, message string, args ...interfac
 
 func (l *fileHandler) emit(ctx *MessageContext, message string, args ...interface{}) {
 	filename := FilePath()
-	originalMessage := fmt.Sprintf(message, args...)
```

Comment:
There is important logic happening here that doesn't seem replicated in the new implementation, namely:

* Only reporting on "public" channels
* Including the most recent logging messages
* Including the command that was ran along with its flags